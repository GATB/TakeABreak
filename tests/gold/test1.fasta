>inv_0 au
AGATAACTACAGTGCCGCTTACAGCCCCTCATAGACCAGTTCATTCCAGATCTCCTTCC
>vb
AGGCTATATTACAGACGTCGGCGACGACAGTGACAGAAACTGTGCAAGTACCTAATCTC
>av'
AGATAACTACAGTGCCGCTTACAGCCCCTCTGTCGTCGCCGACGTCTGTAATATAGCCT
>u'b
GGAAGGAGATCTGGAATGAACTGGTCTATGTGACAGAAACTGTGCAAGTACCTAATCTC
>inv_1 au
AGCATAGCCATTTATCGGAGCGCCTCGGTACACGGTATGACCAGACGCCTCGTGAGACC
>vb
GTTGCGTTCCTAGCGCCTATATTTGTCTCTTTGCCGGCTTATGTGGACAAGCATAGCAT
>av'
AGCATAGCCATTTATCGGAGCGCCTCGGTAGAGACAAATATAGGCGCTAGGAACGCAAC
>u'b
GGTCTCACGAGGCGTCTGGTCATACCGTGTTTGCCGGCTTATGTGGACAAGCATAGCAT
>inv_2 au
AGTGTAGGTTCTGACCGATACGTGCTTCGTGGCGTAGGGCTTATGTCCCCAGTTGTAAAA
>vb
TGTCCCCAGTTGTAAAATTGTGAGTTCTCACATCATCTACTGACGTCCCTGAGGCTGCAG
>av'
AGTGTAGGTTCTGACCGATACGTGCTTCGTTGAGAACTCACAATTTTACAACTGGGGACA
>u'b
TTTTACAACTGGGGACATAAGCCCTACGCCCATCATCTACTGACGTCCCTGAGGCTGCAG
>inv_3 au
ATGGGACAGTATCCGCCGCAAGTTCTAGTGCAATGGCGGTATAGTACGCTCGTACTGTAG
>vb
GTGGGATCATCACTAATAAGGATACTGGGAAGACTCACAGGCCTCCGCCTATAGGCGGTG
>av'
ATGGGACAGTATCCGCCGCAAGTTCTAGTGTCCCAGTATCCTTATTAGTGATGATCCCAC
>u'b
CTACAGTACGAGCGTACTATACCGCCATTGAGACTCACAGGCCTCCGCCTATAGGCGGTG
>inv_4 au
CAAAGCCGCGTGACGACGGCTGTCGGCCCGCACCAGGGCTGTTCAATCGAGGCTCACGGG
>vb
TCACGGGAGCTTATTGCGGGGATTCTGTTAGGCCCCATCAGTAGCCCGCTCGGCCCAGGA
>av'
CAAAGCCGCGTGACGACGGCTGTCGGCCCGTAACAGAATCCCCGCAATAAGCTCCCGTGA
>u'b
CCCGTGAGCCTCGATTGAACAGCCCTGGTGGGCCCCATCAGTAGCCCGCTCGGCCCAGGA
>inv_5 au
CAGAAATCCGCGTGATTACGAGTCGTGGTAAATTTAATCTGGCTGTGGTCTAGACATTCC
>vb
GTCCTGTGAGCAGCGAAGGCCCATACGCGAGATACACTGCCAGAAATCCGCGTGATTACG
>av'
CAGAAATCCGCGTGATTACGAGTCGTGGTATCGCGTATGGGCCTTCGCTGCTCACAGGAC
>u'b
GGAATGTCTAGACCACAGCCAGATTAAATTGATACACTGCCAGAAATCCGCGTGATTACG
