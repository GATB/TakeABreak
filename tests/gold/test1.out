TakeABreak                              
    version                                  : 1.1.0
    gatb-core-library                        : 1.1.0
Parameters                              
    Input data                              
        Reads                                    : ../data/toy_example_reads.fasta,../data/toy_example_with_inv_reads.fasta
    Graph                                   
        kmer-size                                : 31
        abundance                                : 3
        nb_solid_kmers                           : 1319
        nb_branching_nodes                       : 26
    Inversions                              
        repeat                                   : 8
        max_sim                                  : 80
        LCT                                      : 100
Results                                 
    nb_occurrences                           : 24
    nb_distinct_solutions                    : 6
    time                                     : 0.0 s
    output_files                            
        graph_file                               : newtests/test1.h5
        solution_file                            : newtests/test1.fasta
